{"name": "TC05: Delete users", "status": "broken", "statusDetails": {"message": "TypeError: WebDriver.__init__() got an unexpected keyword argument 'service'", "trace": "request = <SubRequest 'init_web_driver' for <Function test_verify_login>>\n\n    @pytest.fixture(scope=\"class\")\n    def init_web_driver(request):\n        if get_data(\"ExecuteApplitools\").lower() == 'yes':\n            globals()['driver'] = get_web_driver()\n        else:\n>           edriver = get_web_driver()\n\nconftest.py:42: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\nconftest.py:96: in get_web_driver\n    driver = get_chrome()\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\n    def get_chrome():\n>       chrome_driver = webdriver.Chrome(service=ChromeService(ChromeDriverManager().install()))  # Selenium 4.x\nE       TypeError: WebDriver.__init__() got an unexpected keyword argument 'service'\n\nconftest.py:125: TypeError"}, "description": "This test delete new users and check that the users are deleted", "start": 1682172172272, "stop": 1682172172272, "uuid": "62f65286-dec5-4588-b203-f594d2eb1079", "historyId": "869147185e4927322c8b087d5d7b7714", "testCaseId": "f17644ba3b640c47456d6409fd6c32e3", "fullName": "test_cases.test_web.TestWeb#test_verify_deleted_user", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('init_web_driver')"}, {"name": "parentSuite", "value": "test_cases"}, {"name": "suite", "value": "test_web"}, {"name": "subSuite", "value": "TestWeb"}, {"name": "host", "value": "BarLaptop"}, {"name": "thread", "value": "20408-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_cases.test_web"}]}